### PR TITLE
Test window.onerror when <script src> URL contains a hash

### PR DESCRIPTION
**Pull request by zcorpan**
See https://github.com/whatwg/html/pull/2075

### CODE REVIEW

**Comment by zcorpan** [comment]
@nhiroki r? 🙂

File: html/webappapis/scripting/processing-model-2/compile-error-same-origin-with-hash.html [hunk 1]
```diff
@@ -0,0 +1,36 @@
+<!doctype html>
+<html>
+ <head>
+  <title>window.onerror - compile error in &lt;script src=...> with hash</title>
```
Comment (nhiroki) [inline html/webappapis/scripting/processing-model-2/compile-error-same-origin-with-hash.html]: Can you replace ```src=...>``` with ```src=...&gt;```?

**Comment by nhiroki** [comment]
LGTM. Thank you :)

File: html/webappapis/scripting/processing-model-2/compile-error-same-origin-with-hash.html (hunk 1 above)
Comment (zcorpan) [inline html/webappapis/scripting/processing-model-2/compile-error-same-origin-with-hash.html]: Why? > does not need escaping in HTML.

Comment (nhiroki) [inline html/webappapis/scripting/processing-model-2/compile-error-same-origin-with-hash.html]: Sorry for my delayed reply. I thought it's more consistent with "&lt;", but as you commented, it's not necessary. I'm ok with leaving it as is.

**Pull request merged**